server/sonar-web/src/main/js/components/source-viewer/main.js

```diff
@@ -274,7 +274,8 @@ define([
               options = {
                 data: {
                   componentUuids: this.model.id,
-                  extra_fields: 'actions,transitions,actionPlanName',
+                  f: 'component,componentId,project,subProject,rule,status,resolution,author,reporter,assignee,debt,line,message,' +
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) The line contains 132 characters which is greater than 120 authorized. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=javascript%3ALineLength)